{
	"schema_version": "1.6.0",
	"id": "MAL-2024-6253",
	"modified": "2024-06-25T13:44:28Z",
	"published": "2024-06-25T13:44:28Z",
	"summary": "Malicious code in zatta (PyPI)",
	"details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
	"affected": [
		{
			"package": {
				"ecosystem": "PyPI",
				"name": "zatta",
				"purl": "pkg:pypi/zatta"
			},
			"versions": [
				"1"
			],
			"database_specific": {
				"source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/zatta/MAL-2024-6253.json"
			}
		}
	],
	"credits": [
		{
			"name": "ReversingLabs",
			"contact": [
				"https://www.reversinglabs.com"
			],
			"type": "FINDER"
		}
	],
	"database_specific": {
		"malicious-packages-origins": [
			{
				"id": "RLMA-2024-05057",
				"import_time": "2024-06-28T02:51:14.72235761Z",
				"modified_time": "2024-06-25T13:44:28Z",
				"sha256": "efbebd4745ad14d5a115f706c1d5f429aad05c8fa2d29ca72fa0722ef13bec1b",
				"source": "reversing-labs",
				"versions": [
					"1"
				]
			}
		]
	}
}
